Mono.Debugging.Soft/PortablePdbData.cs

```diff
@@ -62,12 +59,18 @@ public static bool IsPortablePdb (string pdbFileName)
 			}
 		}
 
-		private string pdbFileName;
+
+		readonly byte [] pdbBytes;
+		private readonly string pdbFileName;
 
 		public PortablePdbData (string pdbFileName)
 		{
 			this.pdbFileName = pdbFileName;
```

Comment:
This will handle some exceptions on invalid paths, helping us handle invalid data better.

```suggestion
			this.pdbFileName = Path.GetFullPath (pdbFileName);
```

Reply:
These are now needed, as we have 2 constructors